Mathlib/Logic/Basic.lean

```diff
@@ -560,9 +560,17 @@ theorem eqRec_heq' {α : Sort u_1} {a' : α} {motive : (a : α) → a' = a → S
   by subst t; rfl
 
 theorem rec_heq_of_heq {C : α → Sort _} {x : C a} {y : β} (e : a = b) (h : HEq x y) :
-    HEq (@Eq.ndrec α a C x b e) y := by subst e; exact h
+    HEq (e ▸ x) y := by subst e; exact h
```

Comment:
In mathlib 3, this theorem is always used in a context where `e ▸ x` will work. I changed it for readability, but if we want `Eq.ndrec` specifically for some reason I'll change it back.